generator/flat/templates/record.go

```diff
@@ -22,7 +22,7 @@ type {{ .Name }} struct {
 {{ end }}
 }
 
-const {{ .Name }}AvroCRC64Fingerprint = "{{ definitionFingerprint . }}"
+var {{ .Name }}AvroCRC64Fingerprint = {{ definitionFingerprint . }}
```

Comment:
If we can't make this a const because it's a byte slice, let's just put the value inside the method definition? Right now a user could reassign the fingerprint variable for a generated struct which would lead to weird behaviour.

Reply:
See my comment about the use of %q